transformations/transformations/utility_routines.py

```diff
@@ -64,6 +64,17 @@ def transform_subroutine(self, routine, **kwargs):
 
         routine.body = Transformer(mapper).visit(routine.body)
 
+        #Get rid of unused import and variable
+        if self.remove:
+            for imp in FindNodes(Import).visit(routine.spec):
+                if 'LHOOK' in [s.name for s in imp.symbols]:
```

Comment:
It should be sufficient to test `if 'LHOOK' in imp.symbols`, since this will also take care of making the comparison case-insensitive.